test/components/bbox.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:test/test.dart';
+import 'package:turf/bbox.dart';
+import 'package:turf/helpers.dart';
+
+main() {
+  test('bbox', () {
```

Comment:
In the original turf.js package, they cover many more different testing scenarios, e.g. different geometry types, collections, recompute, and a null-geometry in a feature. 

https://github.com/Turfjs/turf/blob/master/packages/turf-bbox/test.js

Wdyt, should we cover some more testing cases?